### PR TITLE
Full-height infobox on desktop

### PR DESCRIPTION
**Pull request by danielcolinjames**
ezgif-2-f869a6d77563

### CODE REVIEW

**Comment by danielcolinjames** [comment]
related to #498, I think this might help a little bit with the feeling that the infobox is "cramped" or too small on desktop. and the width could be extended a little bit on top of the full height change, since there wouldn't be a margin on the left of the infobox any more with this change

this might be the wrong direction, but wanted to try it out and see how it felt on desktop like this

I still don't think allowing for maximizing the _width_ of the infobox is necessary, and I think doing that will inherently force us to make views that aren't mobile-friendly

would be curious to hear your thoughts @PeterMain

**Comment by PeterMain** [comment]
I agree on your points. I've yet to hear a good reason why we would allow for wider info box widths (I guess our current widgets could flex to some 3 per row breakpoint?). The concern there is that it introduces structures that won't collapse as easily. Maybe thats the point though? A larger screen size could present higher fidelity widgets and list metadata.

As for full height, I did some mocking up and arrived at the same solution as you. Fixed height for top tinted area (with sufficient top padding), and then use the remaining height for info box content.

Full Height Info box

**Comment by cokes518** [comment]
:shipit:

**Comment by cokes518** [comment]
noticed the map box attribution font size increases when you close the infobox, but does not resize when you reopen the infobox. this is not a blocker so we can ship this neat change.

File: components/InfoBox/InfoBox.js [hunk 1]
```diff
@@ -55,21 +54,38 @@ const InfoBox = ({ title, metaTitle, children, breadcrumbs, subtitles }) => {
...
             )}
           >
             {children}
           </div>
         </div>
+        <div
+          className="fixed hidden md:flex pointer-events-auto cursor-pointer right-0 top-28 button-blur items-center justify-center h-14 w-5 rounded-r-md -mr-5"
+          onClick={toggleInfoBox}
+        >
+          <svg
```
Comment (allenan) [inline components/InfoBox/InfoBox.js]: can we extract this svg into an icon component?

**Review by allenan — APPROVED** [review]
let's give this a shot! just one comment about the svg but then this can merge

File: components/InfoBox/InfoBox.js (hunk 1 above)
Comment (danielcolinjames) [reply components/InfoBox/InfoBox.js]: good call, we actually already had a ChevronIcon component, so now it reuses that 👍

**Pull request merged**